Makefile

```diff
@@ -141,6 +153,52 @@ liccheck:
 copyright-check:
 	python scripts/check_copyright.py
 
+####################
+### Clean and init commands
+####################
+
+.PHONY: rmcache
+rmcache:
+	find . | grep -E "\(__pycache__|\.pyc|\.pyo$)" | xargs rm -rf
```

Comment:
Could you please mirror what we have in the aea framework?
Missing many of the same elements here.